packages/vscode-extension/package.nls.json

```diff
@@ -458,5 +458,5 @@
     "teamstoolkit.chatParticipants.create.selectFolder.label": "Select Folder",
     "teamstoolkit.chatParticipants.create.successfullyCreated": "Project successfully created in current workspace.",
     "teamstoolkit.chatParticipants.create.failToCreate": "Unable to create the project.",
-    "teamstoolkit.chatParticipants.default.noConceptualAnswer": "I can only answer conceptual questions."
```

Comment:
Proposed: I can only answer basic questions about Teams app development. Try these commands or get more info from [Teams Toolkit documentation](https://learn.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-fundamentals).

Reply:
thanks @supkasar , this string is updated now.